docs/acme-integration.rst

```diff
@@ -153,6 +153,33 @@ configuration active, it will check for validity of the signed certificate, and
 about a month before the expiration date it will try to renew the certificate
 automatically.
 
+I want a certifcate for subdomains but domain
+---------------------------------------------
+
+Yes, it's possible :-) Please consult the example and create your own similar
+configuration. In the example we create a certificate for ``logs.it-zone.org``
+and ``mon.it-zone.org`` subdoimains, without creating cert for ``it-zone.org``
```

Comment:
Perhaps use the `example.com` or `example.org` domain instead of a custom domain. These are reserved for documentation.